nimble/controller/src/ble_ll_sched.c

```diff
@@ -920,6 +920,11 @@ ble_ll_sched_rmv_elem_type(uint8_t type, sched_remove_cb_func remove_cb)
     OS_ENTER_CRITICAL(sr);
 
     first = TAILQ_FIRST(&g_ble_ll_sched_q);
+    if (!first) {
+        OS_EXIT_CRITICAL(sr);
+        return;
+    }
+
     if (first->sched_type == type) {
         first_removed = 1;
     }
```

Comment:
Please note that this shall be fixed as well as `first_removed` can be left uninitialized.